dm/relay/local_reader_test.go

```diff
@@ -613,7 +613,7 @@ func (t *testReaderSuite) TestStartSyncByGTID(c *C) {
 		{
 			"ba8f633f-1f15-11eb-b1c7-0242ac110002",
 			"ba8f633f-1f15-11eb-b1c7-0242ac110002.000001",
-			"ba8f633f-1f15-11eb-b1c7-0242ac110002:0",
+			"ba8f633f-1f15-11eb-b1c7-0242ac110002:1",
```

Comment:
Should we just add a test case rather than replace it? Because it seems `IsNilMySQLGTIDSet` is used to handle such cases (i.e. xxx:0).

Reply:
This gtid is used to generate binlog event, after this pr, xxx:0 will be transformed to nil, but binlog pkg could not generate event with nil gtid.